auth_backend/backends.py

```diff
@@ -34,10 +34,25 @@ def authenticate(self, email=None, username=None, password=None, **kwargs):
         if strategy:
             payload['strategy'] = strategy
 
+        auth_api_client = AuthApiClient(cas_credentials)
         status, data = auth_api_client.call('sessions', 'POST', payload)
 
         if status not in (200, 404,):
```

Comment:
`(200, 404)` is clearly a tuple

the only time you need a trailing `,` is when the tuple has one value.